Core/Core/Grades/GradesViewController.swift

```diff
@@ -135,6 +136,17 @@ public class GradesViewController: UIViewController {
             view.setNeedsLayout()
         }
     }
+
```

Comment:
I find putting in an extension helps make it clearer that this comes from some protocol
```
extension GradesViewController: HorizontalPagedMenuItem {
    public func itemWillBeDisplayed() 
...
}
```

Again this is a knitpick.  Do it if you like

Reply:
No problem with that. I'll do it.